pidcat.py

```diff
@@ -41,10 +44,16 @@
 parser.add_argument('-c', '--clear', dest='clear_logcat', action='store_true', help='Clear the entire log before running.')
 parser.add_argument('-t', '--tag', dest='tag', action='append', help='Filter output by specified tag(s)')
 parser.add_argument('-i', '--ignore-tag', dest='ignored_tag', action='append', help='Filter output by ignoring specified tag(s)')
+parser.add_argument('--verbose', dest='verbose', action='store_true', help='Shows all logcat lines and some script debub info')
 
 args = parser.parse_args()
 min_level = LOG_LEVELS_MAP[args.min_level.upper()]
 
+VERBOSE = args.verbose
+
+if not args.package :
+  print "Warning: No package name provided"
```

Comment:
OK, I'll add that tomorrow morning. 
-MIke 
Sent using my Android phone. 
Please forgive my brevity. 
On Feb 18, 2015 12:14 AM, Jake Wharton <notifications@github.com> wrote:In pidcat.py:
>  
>  args = parser.parse_args()
>  min_level = LOG_LEVELS_MAP[args.min_level.upper()]
>  
> +VERBOSE = args.verbose
> +
> +if not args.package :
> +  print "Warning: No package name provided"

need parenthesis for python 3

—Reply to this email directly or view it on GitHub.